web-admin/src/routes/[organization]/[project]/reports/[report]/download/+page.svelte

```diff
@@ -0,0 +1,58 @@
+<script lang="ts">
+  import { goto } from "$app/navigation";
+  import { page } from "$app/stores";
+  import { createDownloadReportMutation } from "@rilldata/web-admin/features/projects/download-report";
+  import CtaButton from "@rilldata/web-common/components/calls-to-action/CTAButton.svelte";
+  import CtaContentContainer from "@rilldata/web-common/components/calls-to-action/CTAContentContainer.svelte";
+  import CtaLayoutContainer from "@rilldata/web-common/components/calls-to-action/CTALayoutContainer.svelte";
+  import CtaMessage from "@rilldata/web-common/components/calls-to-action/CTAMessage.svelte";
+  import type { V1ExportFormat } from "@rilldata/web-common/runtime-client";
+  import { runtime } from "@rilldata/web-common/runtime-client/runtime-store";
+
+  $: reportId = $page.params.report;
+  $: format = $page.url.searchParams.get("format");
+  $: bakedQuery = $page.url.searchParams.get("query");
+  $: limit = $page.url.searchParams.get("limit");
+
+  const downloadReportMutation = createDownloadReportMutation();
+  let once = false;
```

Comment:
Agree with CodeRabbit! `mutationRunOnce` would be more clear

Reply:
Sure. I am naming it `downloadOnce` instead. `mutationRunOnce` is a little too generic.